tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/CrashSnapshotReporter.cs

```diff
@@ -79,13 +62,12 @@ public async Task EndCaptureAsync (TimeSpan timeout)
 				newCrashFiles.ExceptWith (initialCrashes);
 
 				if (newCrashFiles.Count == 0) {
-					if (stopwatch.Elapsed.TotalSeconds > timeout.TotalSeconds) {
-						break;
-					} else {
+					if (stopwatch.Elapsed.TotalSeconds > timeout.TotalSeconds) break;
```

Comment:
Is there some sort of auto formatting happening? Because this is formatting to the wrong style...

Reply:
Yeah.. Visual Studio did that.. I will revert this

Reply:
I did revert this but it doesn't matter much because this code will be moved to `dotnet/xharness` and new formatting (.NET) will be applied there anyway.